client/test/test-server.js

```diff
@@ -35,7 +35,12 @@ arc.post('/reset', function(req, res, next) {
   }
 
   var modelsToReset = workspace.models().filter(function(m) {
-    return m !== 'PackageDefinition' && m !== 'Facet' && m !== 'ConfigFile';
+    return [
```

Comment:
My preference would be to use something other than an Array search for multiple choice checks like this (because JS Array searches don't convey the intent like it does in some other langages), but I'll defer style preferences to the people spending more time in the Arc code base.

```js
return /^(PackageDefinition|Facet|ConfigFile|Workspace)$/.test(m.modelName);
```

```js
switch (m.modelName) {
  case 'PackageDefinition':
  case 'Facet':
  case 'ConfigFile':
  case 'Workspace':
    return true;
  default:
    return false;
}
```

```js
return {
  PackageDefinition: true,
  Facet: true,
  ConfigFile: true,
  Workspace: true,
}[m.modelName] || false;
```